src/libraries/Microsoft.PowerFx.Interpreter/Utility.cs

```diff
@@ -30,5 +35,34 @@ public static T GetService<T>(this IServiceProvider serviceProvider)
         {
             return (T)serviceProvider.GetService(typeof(T));
         }
+
+        public static FormulaValue SetUntypedObject(IRContext context, UntypedObjectBase untypedObject, FormulaValue property, FormulaValue value)
```

Comment:
nit - this could be an extension method on untypedObject. That would make discoverability a bit easier

Reply:
Good suggestion. Done.